Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV GRADLE_VERSION 4.10.3
 ENV RUBY_VERSION 2.6.1
 ENV MIX_VERSION 1.0
 ENV JDK_VERISON 8u211
+ENV COMPOSER_ALLOW_SUPERUSER 1
```

Comment:
Curious as to why this is needed?

Reply:
There's a warning that gets outputted on the docker image that causes invalid json. Adding that silences the composer warning.